src/libs/antares/study/parts/thermal/cluster.cpp

```diff
@@ -608,6 +623,36 @@ bool Data::ThermalCluster::integrityCheck()
             ret = false;
     }*/
 
+    // ramping
+    if (maxUpwardPowerRampingRate <= 0)
+    {
+        logs.error() << "Thermal cluster: " << parentArea->name << '/' << pName
+                     << ": The maximum upward power ramping rate must greater than zero.";
+        maxUpwardPowerRampingRate = 1.;
```

Comment:
i would be helpful to print the fallback value;
also it seems that the return value is never used

Reply:
Now, with commit [8133150](https://github.com/AntaresSimulatorTeam/Antares_Simulator/pull/1817/commits/8133150b562b9e7655a86f76893116f5b9e5ccea), the ramping is disabled in this case, and this information is printed.